### ISSUE TITLE
tmuxinator replay-pane -t ''

### ISSUE THREAD

**Issue by emayssat-ms** [tmuxinator/tmuxinator#473]
it would be great to have a command line that can restore the configuration of a pane given the configured commands.
In other word, I would like to restore settings/configuration while still in a tmux session.

tmux replay-window -n 'window_name' -t ''
to replay the window_name configuration in the current window.

Thanks

**Comment by adamstrickland** [comment]
@emayssat-ms can you take a look at https://github.com/tmuxinator/tmuxinator/issues/61? I think it covers what you're suggesting here.

**Comment by emayssat-ms** [comment]
No. This is different from #61. +1 for  #61 though! This case is not about generating a tmuxinator configuration file from a tmux session. This case is about executing the directives from an existing file into the current tmux session. For example, panes that were close needs to be reopen. Command in pane are reexecuted. This is why it is a 'replay'.

In other words, I want a configuration to be a reference state to which I can quickly go back to if desired.
